bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        use_stdin=True,
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+): (?P<message>.*)')
+class TOMLBear:
+    """
+    This bear validates TOML documents and prints each key's type.
```

Comment:
you should also add a link, to where more information about tomlv can be fetched